packages/oc-template-es6-compiler/lib/to-abstract-base-template-utils/oc-webpack/lib/configurator/index.js

```diff
@@ -1,102 +1,164 @@
 'use strict';
 
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const MinifyPlugin = require('babel-minify-webpack-plugin');
 const path = require('path');
+const loaderUtils = require('loader-utils');
+const postcssNormalize = require('postcss-normalize');
+const TerserPlugin = require('terser-webpack-plugin');
 const webpack = require('webpack');
 
 const createExcludeRegex = require('./createExcludeRegex');
 
+function getLocalIdent(context, localIdentName, localName, options) {
```

Comment:
I think I'm missing some context behind introducing this function. I see that previously it was a variable. Why do we need such extensive change here?

Reply:
Because I updated CSS loader from 0.x to 6.x and that has introduced a lot of breaking changes, including the fact that know this needs to be a function.